src/Provider/Intercom.php

```diff
@@ -119,6 +125,10 @@ protected function fetchResourceOwnerDetails(AccessToken $token)
      */
     protected function createResourceOwner(array $response, AccessToken $token)
     {
-        return new IntercomResourceOwner($response);
+        $validated_response = $response;
```

Comment:
To meet PSR2 standard, I would use camel case for the variable names (`$validatedResponse`, `$this->verifyEmail`

Reply:
Thanks! I'll update the name :)